docs/reference/components/transforms/remap.cue

```diff
@@ -41,8 +41,10 @@ components: transforms: remap: {
 			type: string: {
 				examples: [
 					"""
-						.type = "foo",
-						.new_field = .old_field * 2
+						. = parse_json(.message)
```

Comment:
FWIW I think users will generally want to use `. = merge(., parse_json(.message))` to avoid clobbering any existing metadata fields.